src/routes/docs/tutorials/nextjs/step-2/+page.markdoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: tutorial
+title: Create Next.js app
+description: Create a Next.js project using Appwrite.
+step: 2
+---
+
+# Create Next.js project {% #create-next-js-project %}
+
+Create a new Next.js project using the `create-next-app` command.
+
+```sh
+npx create-next-app expense-tracker --eslint
```

Comment:
What are the options you plan to take?

for example ```
✔ Would you like to use TypeScript? … No / Yes
✔ Would you like to use Tailwind CSS? … No / Yes
✔ Would you like to use `src/` directory? … No / Yes
✔ Would you like to use App Router? (recommended) … No / Yes
✔ Would you like to customize the default import alias (@/*)? … No / Yes
Creating a new Next.js app in /Users/wen/Documents/GitHub/expense-tracker.
```

smthing like this might be helpful